README.md

```diff
@@ -834,8 +834,9 @@ __Arguments__
 * `fn(callback)` - A function which is called each time `test` passes. The function is
   passed a `callback(err)`, which must be called once it has completed with an
   optional `err` argument.
-* `callback(err)` - A callback which is called after the test fails and repeated
-  execution of `fn` has stopped.
+* `callback(err, [results])` - A callback which is called after the test fails
+  and repeated execution of `fn` has stopped. This will be passed the results
+  of the last call to `fn`'s callback.
```

Comment:
> A callback which is called after the test fails and repeated execution of `fn` has stopped. This will be passed the results of the last call to `fn`'s callback.

This doesn't read so well, maybe
> A callback which is called after the test function has failed and repeated execution of `fn` has stopped. The `callback` will be passed an error and any arguments passed to the final `fn`'s callback.